PulseSampleApp/app/build.gradle

```diff
@@ -0,0 +1,70 @@
+apply plugin: 'com.android.application'
+
+android {
+    buildToolsVersion "21.1.2"
+    compileSdkVersion 23
+
+    defaultConfig {
+        applicationId "com.ooyala.sample.PulseSampleApp"
+        minSdkVersion 16
+        targetSdkVersion 21
+
+        ndk {
+            // React Native for Android is incompatible with 3rd-party 64-bit libraries.
+            abiFilters "armeabi-v7a", "x86"
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+        }
+    }
+}
+
+
+task copyAssetsTask1(type: Copy) {
```

Comment:
Those libs are just symlinks to the files in vendors. Do you want me to remove the symlinks and use copies instead?